### ISSUE TITLE
Unstack a window by dragging

### ISSUE THREAD

**Issue by blahajjessie** [pop-os/cosmic-epoch#396]
I have been a long time user of Pop Shell (the gnome extension) and found it rather odd that I was not able to drag a window out off a stack to remove it from the stack. It seems like the only way to do this at the moment is to right click and press "Unstack window"
Honestly, I'm not sure if this is a feature request or a bug but I'm assuming that it's a bug. If it is  a feature request or is in the pipeline, feel free to let me know. 

As a sidenote, when the user is focused on a stack, windows only add to the stack. I don't think this is the case in the gnome extension and it's rather annoying. This seems like it could be a  feature though.

**Comment by thedeanominator** [comment]
I think this issue has been resolved, at least in the latest version of the COSMIC desktop (PopOS 24.04) where you can drag an application outside of a stack using the mouse and into its own tile.

**Comment by jacobgkau** [comment]
Yeah, you can unstack a window with the mouse by dragging the window's tab out of the stack.
